superset-frontend/src/pages/ExecutionLogList/index.tsx

```diff
@@ -103,10 +103,18 @@ function ExecutionLog({ addDangerToast, isReportEnabled }: ExecutionLogProps) {
           row: {
             original: { uuid: executionId },
           },
-        }: any) => (executionId ? executionId.slice(0, 6) : 'none'),
+        }: any) => (
+          <div
```

Comment:
[optional] this might look cute as an antd <Label>, could also add a monospace bool prop (default false) to the component